e2e/actions/oci.go

```diff
@@ -1969,6 +1981,24 @@ func (c actionTests) actionOciNoCompat(t *testing.T) {
 			args:     []string{"--no-compat", imageRef, "sh", "-c", "ls /dev/block"},
 			exitCode: 0,
 		},
+		// CWD in container is host CWD
+		{
+			name:     "cwd",
+			args:     []string{"--no-compat", imageRef, "sh", "-c", "pwd"},
+			exitCode: 0,
+			expect: []e2e.SingularityCmdResultOp{
+				e2e.ExpectOutput(e2e.ContainMatch, workDir),
+			},
+		},
+		// CWD is bind mounted from host
+		{
+			name:     "cwdBind",
+			args:     []string{"--no-compat", imageRef, "mount"},
```

Comment:
Checking that cwd is not mounted when it should have been by making sure the output of `mount` doesn't contain what it shouldn't contain makes sense to me; but don't we usually check for successful bind-mounts using the "canary" pattern...?